tests/integration/states/file.py

```diff
@@ -332,14 +391,16 @@ def test_managed_file_with_gitpillar_sls(self):
         file is rendered properly and is created.
         '''
         state_name = 'file-pillargit'
+
         ret = self.run_function('state.sls', [state_name])
         self.assertSaltTrueReturn(ret)
 
         #Check to make sure the file was created
         check_file = self.run_function('file.file_exists', [FILEPILLARGIT])
         self.assertTrue(check_file)
 
-    @skipIf(os.geteuid() != 0, 'you must be root to run this test')
+    @skipIf(not IS_ADMIN, 'you must be root to run this test')
+    @skipIf(not HAS_PWD, "pwd not available. Skipping test")
```

Comment:
Same thing is above. We need to test for a clean error if deps aren't met instead of just leave it totally untested.